--- "Werewolf for Telegram/ClearUpdates/Program.cs"
+++ "Werewolf for Telegram/ClearUpdates/Program.cs"
@@ -146,8 +146,8 @@
             {
                 using (var DB = new WWContext())
                 {
-                    var status = DB.BotStatus.First(x => x.Id == 1);
-                    if (status.BotStatus != "Normal")
+                    var status = DB.BotStatus.FirstOrDefault(x => x.Id == 1);
+                    if (status != null && status.BotStatus != "Normal")
                     {
                         if (!dead)
                         {
